### PR TITLE
Generate SMPLX humanoid models + bug fix generate motions

### PR DESCRIPTION
**Pull request by xavierpuigf**
## Motivation and Context

This PR allows to generate new SMPL-X body models to be used in Habitat. It also adds hands to the avatar motion.

## How Has This Been Tested

Ran script and pytest.

## Types of changes

- **\[Development\]** A pull request that add new features to the [habitat-lab](/habitat-lab) task and environment codebase. Development Pull Requests must be small (less that 500 lines of code change), have unit testing, very extensive documentation and examples. These are typically new tasks, environments, sensors, etc... The review process for these Pull Request is longer because these changes will be maintained by our core team of developers, so make sure your changes are easy to understand!

## Checklist

- [X] My code follows the code style of this project.
- [X] I have updated the documentation if required.
- [X] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [X] I have added tests to cover my changes if required.

### CODE REVIEW

File: scripts/export_smplx_bodies.py [hunk 1]
```diff
@@ -0,0 +1,746 @@
...
+)
+BASE_LIMIT_NODE_STR = None
+multiplier = 1
+root_bone = None
+
+
+####### Export GLTF code ######
+
+
+def smplx_export_gltf(filepath):
```
Comment (aclegg3) [inline scripts/export_smplx_bodies.py:49]: Please add docstrings, typehints, etc... to all functions. :)

File: scripts/export_smplx_bodies.py [hunk 2]
```diff
@@ -0,0 +1,746 @@
...
+    )
+    bpy.types.EditBone.ixz = bpy.props.FloatProperty(
+        name="Inertia value XZ", default=0.0
+    )
+    bpy.types.EditBone.iyz = bpy.props.FloatProperty(
+        name="Inertia value YZ", default=0.0
+    )
+
+
+def main():
```
Comment (aclegg3) [inline scripts/export_smplx_bodies.py:605]: Some comment here about how to run this from CLI w/ blender and linking to the README would be helpful.

**Review by aclegg3 — APPROVED** [review]
Overall this looks good, thanks for adding the script and details. :+1:

**Comment by 0mdc** [comment]
Belated LGTM!

**Pull request merged**